client/src/components/Members/Community/ElcMembersList.js

```diff
@@ -2,19 +2,20 @@ export const ElcMembersLists = [
 
       {
             id: 0,
-            img: './images/Members/Core Electronics/vipul.jpg',
-            name: 'Vipul Pandey',
-            position: 'Community Head',
-            mail: 'https://www.linkedin.com/in/vipul-pandey-3b0728192',
-            linkedin: 'https://www.linkedin.com/in/vipul-pandey-3b0728192'
-      },
-      {
-            id: 1,
-            img: './images/Members/Core Electronics/devansh.jfif',
-            name: 'Devansh Srivastava',
+            img: './images/Members/CoreElectronics/SHASHWAT-PRATAP-COREELECTRONICS.jpg',
+            name: 'Shashwat Pratap',
             position: 'Community Co-ordinator',
-            mail: 'devansh.btech.ec18@iiitranchi.ac.in',
-            linkedin: 'https://www.linkedin.com/in/devansh-srivastava-958766167'
+            mail: 'shaswat.btech.ec19@iiitranchi.ac.in',
+            linkedin: 'https://www.linkedin.com/in/shashwat-pratap-470b7116b',
+            github: 'https://github.com/shashwat0426'
       }
+      // ,{
+      //       id: 1,
+      //       img: './images/Members/Core Electronics/devansh.jfif',
+      //       name: 'Devansh Srivastava',

```

Comment:
Devansh Srivastava is not holding any posts in House of Geeks anymore. I think you should remove this segment.